.bazelrc

```diff
@@ -1,18 +1,15 @@
 # NOTE: this workspace Bazel config was copied from proxy-wasm-cpp-host.
 # Consider moving copts and linkopts into BUILD rules upstream.
 
-# Pass CC, CXX and PATH from the environment.
-build --action_env=CC
-build --action_env=CXX
-build --action_env=PATH
-
+# Service Extensions uses v8.
```

Comment:
Decision subject to change (and leaks implementation details). Maybe say "pick an engine to invoke sample plugins" or something similar.